clang/lib/CConv/Utils.cpp

```diff
@@ -366,4 +366,9 @@ const TypeVariableType *getTypeVariableType(DeclaratorDecl *Decl){
     }
   }
   return nullptr;
+}
+
+bool isTypeAnonymous(QualType T) {
```

Comment:
In what sense are these types "anonymous" ? When I saw the call to this function above I didn't immediately see what is was for, from context.

Reply:
They're anonymous in that they don't have a name that can be use when outputting them as a type argument.